src/content/en/updates/2017/01/scrolling-intervention.md

```diff
@@ -0,0 +1,150 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: We know that scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems.
+
+{# wf_updated_on: 2017-01-10 #}
+{# wf_published_on: 2017-01-10 #}
+{# wf_tags: intervention,chrome56 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: We know that scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems. #}
+
+# Making touch scrolling fast by default {: .page-title }
+
+{% include "web/_shared/contributors/dtapuska.html" %}
+
+We know that scrolling responsiveness is critical to the user's engagement with
+a website on mobile, yet touch event listeners often cause serious scrolling
+performance problems. Chrome has been addressing this by allowing touch event
+listeners to be
+[passive](https://developers.google.com/web/updates/2016/06/passive-event-listeners?hl=en)
+(passing the `{passive: true}` option to `addEventListener()`) and shipping the
+[pointer
+events](https://developers.google.com/web/updates/2016/10/pointer-events) API.
```

Comment:
Use absolute URLs without the fully qualified domain. Using the fully qualified domain breaks the .cn site.